File: x.js
async function start() {
    try {
        const data = await fetch('https://worldtimeapi.org/api/timezone/America/New_York');
        if (!data.ok) {
            throw new Error('Bad network');
        }
        const result = await data.json();
        console.log(result); // Log result.time instead of data.time
    } catch (error) {
        console.log('ERROR: ', error);
    }
}

start();

File: x.py
import requests

def start():
    try:
        response = requests.get('https://worldtimeapi.org/api/timezone/America/New_York')
        response.raise_for_status()  # Check for HTTP errors
        result = response.json()
        print(result['timezone'])  # Use 'datetime' key from the JSON response
    except requests.exceptions.HTTPError as http_err:
        print(f"HTTP error occurred: {http_err}")
    except Exception as err:
        print(f"Other error occurred: {err}")

start()
